CHANGELOG.md

```diff
@@ -5,7 +5,21 @@ All notable changes to this component are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/compare/v0.6.0...HEAD)
+## [Unreleased](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/compare/v0.7.0...HEAD)
+
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+## [0.7.0](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/tag/v0.7.0)
```

Comment:
A lot has been added. Maybe it is worth to add some description?

I think it is worth to call out that:

1. Work towards stabilization
2. Fallback for the service name.
3. Added support for `OTEL_LOG_LEVEL` and improved logging
4. Resource detectors and configuration 
5. Improved user experience (thanks to the rule engine).

Reply:
Most of them are in added section.

I would like to keep it to communicate new dependencies on OTel/DiagnostiSource